Source/ThirdParty/libwebrtc/Source/webrtc/sdk/objc/components/video_codec/RTCVideoDecoderH264.mm

```diff
@@ -156,6 +190,7 @@ - (NSInteger)decodeData:(const uint8_t *)data
   rtc::ScopedCFTypeRef<CMVideoFormatDescriptionRef> inputFormat =
       rtc::ScopedCF(webrtc::CreateVideoFormatDescription(data, size));
   if (inputFormat) {
+    _reorderSize = webrtc::ComputeH264ReorderSizeFromAnnexB(data, size);
```

Comment:
I think we should check that the first packet we received contains a SPS/PPS NAL rather than assume 0 (you'll get a decode error anyway)